cime_config/acme/machines/template.lt_archive

```diff
@@ -1,177 +1,117 @@
-#! /usr/bin/env perl
-#------------------------------------------------------------------------------
+#!/usr/bin/env python2
+
+"""
+template to create a case short term archiving script
+"""
+
+import sys, os, time
+
+workdir = os.environ.get("PBS_O_WORKDIR")
+if workdir is not None:
+    os.chdir(workdir)
+
+_CIMEROOT = os.environ.get("CIMEROOT")
+if _CIMEROOT is None:
+    raise exc_type("ERROR: must set CIMEROOT environment variable")
+
+_LIBDIR = os.path.join(_CIMEROOT, "scripts", "Tools")
+sys.path.append(_LIBDIR)
+
+from standard_script_setup          import *
```

Comment:
Done. Unfortunately, pylint is not installed in any of our systems. I'll install it on my laptop - but for now I did this manually.